chalicelib/templates/base.html

```diff
@@ -324,7 +324,7 @@ <h5 class="center-element">
             },
             socialButtonStyle: 'big',
             languageDictionary: { title: "Log in" },
-            theme: { logo: favicon },
+            theme: { logo: "{{favicon}}" },
```

Comment:
I'm not sure that Jinja templating works in <script> tags. You should test logging in to see if the icon appears; if not, you could use a `<meta>` tag to pass in `favicon`, similar to what is being done with `fs_domain`